apis/network/v1beta1/securitygroup_types.go

```diff
@@ -200,28 +202,31 @@ type IPPermission struct {
 // A SecurityGroupSpec defines the desired state of a SecurityGroup.
```

Comment:
In the two `TODO`s above, it says the jsontags need to be fixed when we bump the version. Could you do that?